admin/app/controllers/TroubleshooterController.scala

```diff
@@ -30,18 +30,14 @@ class TroubleshooterController(wsClient: WSClient) extends Controller with Loggi
 
     val loadBalancers = LoadBalancer.all.filter(_.testPath.isDefined)
 
-    val thisLoadBalancer = loadBalancers.find(_.project == id).head
+    val thisLoadBalancer = loadBalancers.find(_.project == id).headOption
 
+    val directToLoadBalancer = thisLoadBalancer.map(testOnLoadBalancer(_, testPath, id))
+      .getOrElse(Future.successful(TestFailed("Can find the appropriate loadbalancer")))
```

Comment:
Checks are reported like this
<img width="599" alt="screen shot 2016-09-29 at 12 20 46" src="https://cloud.githubusercontent.com/assets/233326/18951871/3959e840-863f-11e6-9081-7e908835150a.png">